definitions/src/lib.rs

```diff
@@ -374,3 +340,98 @@ pub struct Numeric {
     pub sign: Char,
     pub val: [Char; MAX_NUMERIC_LEN],
 }
+
+#[allow(non_camel_case_types)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, FromPrimitive)]
+pub enum SqlBool {
+    SQL_FALSE = 0,
+    SQL_TRUE,
```

Comment:
[nit] I'm thinking that if we use value very specifically instead of relying on the auto increment, it might help validating the correctness with a script/tool.
But if we managed to get all our values from the C headers translated, this might be throw away work.

I think there aren't that many to change. If it is not too time consuming, can you update the enums?

Reply:
by this do you mean explicitly adding `= 1` to `SQL_TRUE`? if so, it won't be a big change :)

Reply:
Yes, this is what I mean. 
There are other enums with this pattern in that file. And I didn't check closely the other files.